packages/next/src/build/webpack/loaders/metadata/resolve-route-data.ts

```diff
@@ -79,6 +83,43 @@ export function resolveSitemap(data: MetadataRoute.Sitemap): string {
         content += `<image:image>\n<image:loc>${image}</image:loc>\n</image:image>\n`
       }
     }
+    if (item.videos?.length) {
+      for (const video of item.videos) {
+        let videoFields = [
+          `<video:video>`,
+          `<video:title>${video.title}</video:title>`,
+          `<video:thumbnail_loc>${video.thumbnail_loc}</video:thumbnail_loc>`,
+          `<video:description>${video.description}</video:description>`,
+          video.content_loc &&
+            `<video:content_loc>${video.content_loc}</video:content_loc>`,
+          video.player_loc &&
+            `<video:player_loc>${video.player_loc}</video:player_loc>`,
+          video.duration &&
+            `<video:duration>${video.duration}</video:duration>`,
+          video.view_count &&
+            `<video:view_count>${video.view_count}</video:view_count>`,
+          video.tag && `<video:tag>${video.tag}</video:tag>`,
+          video.rating && `<video:rating>${video.rating}</video:rating>`,
+          video.expiration_date &&
+            `<video:expiration_date>${video.expiration_date}</video:expiration_date>`,
+          video.publication_date &&
+            `<video:publication_date>${video.publication_date}</video:publication_date>`,
+          video.family_friendly &&
+            `<video:family_friendly>${video.family_friendly}</video:family_friendly>`,
+          video.requires_subscription &&
+            `<video:requires_subscription>${video.requires_subscription}</video:requires_subscription>`,
+          video.live && `<video:live>${video.live}</video:live>`,
+          video.restriction &&
+            `<video:restriction relationship="${video.restriction.relationship}">${video.restriction.content}</video:restriction>`,
+          video.platform &&
+            `<video:platform relationship="${video.platform.relationship}">${video.platform.content}</video:platform>`,
+          video.uploader &&
+            `<video:uploader${video.uploader.info && ` info="${video.uploader.info}"`}>${video.uploader.content}</video:uploader>`,
+          `</video:video>`,
+        ].filter(Boolean)
+        content += videoFields.join('\n')
```

Comment:
need an extra line break at the end

Reply:
added line break at the end of video tag @huozhi